src/ext2020/models.py

```diff
@@ -65,25 +74,47 @@ def __str__(self):
 
 
 class CommunityTrackEvent(models.Model):
-    venue = models.ForeignKey(Venue, on_delete=models.CASCADE)
+    venue = models.ForeignKey(Venue, on_delete=models.CASCADE, null=True, blank=True)
     talk = BigForeignKey(TalkProposal, on_delete=models.CASCADE, null=True, blank=True)
     sponsored_event = BigForeignKey(SponsoredEvent, on_delete=models.CASCADE, null=True, blank=True)
+    custom_event = models.CharField(max_length=140, blank=True)
     order = models.IntegerField(default=0)
 
+    begin_time = models.ForeignKey(
+        to=Time,
+        blank=True,
+        null=True,
+        related_name='begined_%(class)s_set',
```

Comment:
Began? Begun?

Reply:
The code is copied from else where. I'll make it `beginning` for this one (and `ending` on the other one)